.ci/setup_env_centos.sh

```diff
@@ -57,7 +60,7 @@ echo "Install Build Tools"
 sudo -E yum install -y python pkgconfig zlib-devel
 
 echo "Install Kata Containers Kernel"
-"${cidir}/install_kata_kernel.sh" "latest"
+"${cidir}/install_kata_kernel.sh"
```

Comment:
can you move this to the `.ci/setup.sh` instead? so we do not to call it on every distro setup?